infrahub_sdk/branch.py

```diff
@@ -91,6 +121,10 @@ async def create(
         query = Mutation(mutation="BranchCreate", input_data=input_data, query=MUTATION_QUERY_DATA)
         response = await self.client.execute_graphql(query=query.render(), tracker="mutation-branch-create")
 
+        # Make sure server version is recent enough to support background execution, as previously
```

Comment:
not sure if this is so useful because if the server doesn't support `wait_until_completion` it will blow up when you execute the query 2 lines up

Reply:
As discussed, let's use deprecated `background_execution` as query input field so query does not break, and we will switch to `wait_until_completion` the day `background_execution` is completely removed from the graphql query.